osidb/models.py

```diff
@@ -2733,6 +2747,83 @@ def __str__(self):
         return self.username
 
 
+class FlawDraft(FlawMixin, AlertMixin):
+
+    uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    created_dt = models.DateTimeField(auto_now_add=True)
+    updated_dt = models.DateTimeField(auto_now=True)
+    origin = models.URLField(max_length=2048)
+    content = models.JSONField()
```

Comment:
Could you describe this field?

Reply:
This is basically the equivalent of snippets, will update with a comment